scripts/local/.env

```diff
@@ -0,0 +1,80 @@
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_ACCESS_CONNSTRING=
+PCS_IOTHUB_CONNSTRING=
+
+# -----------------------------------------------------------------------------
+# IoTHub Info can be found in the Azure portal at:
+# {Your IoT Hub} > Endpoints > events
+#
+# Example:
+# PCS_IOTHUBREACT_HUB_NAME=iothub-test123
+# PCS_IOTHUBREACT_HUB_ENDPOINT=Endpoint=sb://iothub-foo-iothub-bar-12345-12345.servicebus.windows.net/;SharedAccessKeyName=iothubowner;SharedAccessKey=12345=
+# PCS_IOTHUBREACT_HUB_PARTITIONS=4
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_HUB_NAME=
+PCS_IOTHUBREACT_HUB_ENDPOINT=
+PCS_IOTHUBREACT_HUB_PARTITIONS=
+
+# -----------------------------------------------------------------------------
+# Storage Account information:
+# {Your storage account} > Access keys
+#
+# Example: 
+# PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT=storagetest123
+# PCS_IOTHUBREACT_AZUREBLOB_KEY=12345...6789==
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT=
+PCS_IOTHUBREACT_AZUREBLOB_KEY=
+
+# -----------------------------------------------------------------------------
+# Storage Endpoint Suffix: 
+# {Your storage account} > Properties > Primary Blob Service Endpoint > value
+# in the URI after the "blob." (eg. `core.windows.net`)
+#
+# Example:
+# PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX=core.windows.net
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX=
+
+# -----------------------------------------------------------------------------
+# DocumentDb Conn String:
+# {Your Document DB} > Keys > Primary Connection String
+# Note: use this value for all 3 environment variables below
+#
+# Example:
+# PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING=AccountEndpoint=https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;
+# PCS_TELEMETRY_DOCUMENTDB_CONNSTRING=AccountEndpoint=https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;
+# PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING=AccountEndpoint=https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;
+# -----------------------------------------------------------------------------
+PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING=
+PCS_TELEMETRY_DOCUMENTDB_CONNSTRING=
+PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING=
+
+# -----------------------------------------------------------------------------
+# Default setup for local development
+# - Auth is disabled for local development
+# - You can allow requests to another domain with the PCS_CORS_WHITELIST, see
+#   the following link for more information:
+#   https://docs.microsoft.com/aspnet/web-api/overview/security/enabling-cross-origin-requests-in-web-api
+# - To add a bing map key, please follow the directions here:
```

Comment:
nit: consider re-ordering the comments to match the order of variables below.

Reply:
should the comments be moved next to the variables?

Reply:
+1 for comments next to variables

Reply:
updated